tool/tctl/common/resource_command.go

```diff
@@ -782,6 +794,12 @@ func (rc *ResourceCommand) Delete(ctx context.Context, client auth.ClientI) (err
 			return trace.Wrap(err)
 		}
 		fmt.Printf("%s '%s/%s' has been deleted\n", types.KindCertAuthority, rc.ref.SubKind, rc.ref.Name)
+	case types.KindInstaller:
+		err := client.DeleteInstaller(ctx)
+		if err != nil {
+			return trace.Wrap(err)
+		}
+		fmt.Printf("%s has been deleted\n", types.KindInstaller)
```

Comment:
What does "delete" do? Doesn't it restore the default installer script? If so, the message should probably say that.